docs/dev/guidelines/client.rst

```diff
@@ -353,6 +353,19 @@ When creating a completely new route you will have to register the new paths in
 		}
 	}
 
+12. Strict Template Check
+===================================
+
+To prevent errors for strict template rule in TypeScript, Artemis uses following approaches.
+
+Use ArtemisTranslatePipe instead of TranslatePipe
+**************************************************
+- placeholder="{{ 'global.form.newpassword.placeholder' | translate }}"
++ placeholder="{{ 'global.form.newpassword.placeholder' | artemisTranslate }}"
```

Comment:
GitHub Actions complains here: `Bullet list ends without a blank line; unexpected unindent.`

Reply:
Fixed it